src/common/util.cpp

```diff
@@ -941,12 +941,7 @@ std::string get_nix_version_display_string()
 
   bool sha256sum(const uint8_t *data, size_t len, crypto::hash &hash)
   {
-    SHA256_CTX ctx;
-    if (!SHA256_Init(&ctx))
-      return false;
-    if (!SHA256_Update(&ctx, data, len))
-      return false;
-    if (!SHA256_Final((unsigned char*)hash.data, &ctx))
+    if (!EVP_Digest(data, len, (unsigned char*) hash.data, NULL, EVP_sha256(), NULL))
```

Comment:
for a oneline if/else like this, you should just do:
```return EVP_Digest(...) != 0;
```

Reply:
force pushed it wrong, fixing it currently